pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -229,20 +229,3 @@ func hasListenerMode(l v1beta1.Listener, mode string) bool {
 func GatewaySA(gw *v1beta1.Gateway) string {
 	return model.GetOrDefault(gw.GetAnnotations()[annotation.GatewayServiceAccount.Name], fmt.Sprintf("%s-%s", gw.Name, gw.Spec.GatewayClassName))
 }
-
```

Comment:
Why did this get moved? Import cycles?

Reply:
I got a lint error from the pipelines and git applied the suggested diff.